readme.txt

```diff
@@ -564,12 +564,6 @@ In: class-wp-document-revisions.php
 
 Allows the document file extension to be manipulated.
 
-== Filter document_help ==
```

Comment:
It's been a while, but looking at https://github.com/wp-document-revisions/wp-document-revisions/blob/master/script/build-readme, I believe we should update the `README.md` file with these documentation changes. Once this merges, I can run the `build-readme` script which should combine the various files that makeup the WordPress Plugin Directory readme, and adapt it to the required format.

Reply:
I only recently noticed that the readme.txt had similar text to readme.md and filters,md at the bottom but I had not updated the readme.txt
I have been trying to update them twice with the readme.md being (mainly) updated with the code changes,
Frankly I did not read the build-readme and thus that there was a primary source and a derived one.
Sorry I did not pay enough attention. Clearly I need to take a look here.
Very happy that I would not need to update the readme.txt.